sdcm/utils/common.py

```diff
@@ -899,7 +927,7 @@ def filter_gce_by_tags(tags_dict, instances):
     return filtered_instances
 
 
-def list_instances_gce(tags_dict=None, running=False, verbose=False):
+def list_instances_gce(tags_dict=None, running=False, verbose=False) -> List[Node]:
```

Comment:
Since we run on Python 3.9 it can be just `list[Node]`

Reply:
Fixed